.github/workflows/generate-docs.yml

```diff
@@ -55,31 +55,29 @@ jobs:
         with:
           fetch-depth: 0
           persist-credentials: false
-      - name: Build dpctl
-        if: ${{ !github.event.pull_request || github.event.action != 'closed' }}
-        shell: bash -l {0}
-        run: |
-          source /opt/intel/oneapi/setvars.sh
-          python setup.py develop -- -DCMAKE_C_COMPILER:PATH=icx -DCMAKE_CXX_COMPILER:PATH=icpx -DDPCTL_ENABLE_LO_PROGRAM_CREATION=ON -DDPCTL_DPCPP_HOME_DIR=${BUILD_PREFIX}
-          python -c "import dpctl; print(dpctl.__version__)" || exit 1
-      - name: Build docs
+      - name: Build dpctl+docs
         if: ${{ !github.event.pull_request || github.event.action != 'closed' }}
         shell: bash -l {0}
         run: |
           # Ensure that SYCL libraries are on LD_LIBRARY_PATH
           source /opt/intel/oneapi/setvars.sh
-          cd docs
-          mkdir -p build && cd build && rm -rf *
           wget https://github.com/vovkos/doxyrest/releases/download/doxyrest-2.1.2/doxyrest-2.1.2-linux-amd64.tar.xz
           tar xf doxyrest-2.1.2-linux-amd64.tar.xz
-          cmake .. -DDPCTL_USE_MULTIVERSION_TEMPLATE=ON \
-                   -DDPCTL_ENABLE_DOXYREST=ON \
-                   -DDoxyrest_DIR=`pwd`/doxyrest-2.1.2-linux-amd64
-          make Sphinx || exit 1
-          cd ..
-          mv generated_docs/docs ~/docs
+          python setup.py develop --                       \
+              -G "Unix Makefiles"                          \
+              -DCMAKE_BUILD_TYPE=Debug                     \
+              -DCMAKE_C_COMPILER:PATH=icx                  \
+              -DCMAKE_CXX_COMPILER:PATH=icpx               \
+              -DDPCTL_ENABLE_LO_PROGRAM_CREATION=ON        \
+              -DDPCTL_GENERATE_DOCS=ON                     \
+              -DDPCTL_ENABLE_DOXYREST=ON                   \
+              -DDoxyrest_DIR=`pwd`/doxyrest-2.1.2-linux-amd64
+          python -c "import dpctl; print(dpctl.__version__)" || exit 1
+          pushd "$(find _skbuild -name cmake-build)" || exit 1
+          cmake --build . --target Sphinx || exit 1
+          mv ../cmake-install/docs/docs ~/docs
           git clean -dfx
-          cd ..
+          popd
```

Comment:
I suppose this `popd` is not really needed.

Reply:
done